test/Word/CamelCaseToSeparatorTest.php

```diff
@@ -24,7 +24,7 @@ public function testFilterSeparatesCamelCasedWordsWithSpacesByDefault(): void
     public function testFilterSeparatesCamelCasedWordsWithProvidedSeparator(): void
```

Comment:
If possible, set up a data provider to match a decent set of cases and expected outcomes including fixes for #10 such as:

- 123LeadingNumber
- Number12InTheMiddle
- ANumberAtTheEnd42
- SomePDFFile
- Has-ExistingSeparator
- What_Happens_Here
- leadingLowerCase
- Maybe Others!